### DISCUSSION TITLE
컴포넌트 요구사항 문서화

### DISCUSSION THREAD

**Discussion by nanana3679** [kookmin-sw/capstone-2024-18#13] [Ideas]
CustomButton에서 반복적으로 기능을 수정하는 상황이 발생했습니다.

컴포넌트의 요구사항을 목업과 프로토타입에서 가시화했지만 문서화하지는 않았기 때문이라고 생각합니다. 특히 동적인 간격에 대한 부분은 프로토타입에서도 확인하기 어려운 부분이 있습니다.

따라서 앞으로 컴포넌트를 개발하기 전에 이슈를 등록하여 다음 사항을 명시할 것을 제안합니다.

### 컴포넌트 요구사항
- 컴포넌트의 인자
    - Style을 인자로 받을 경우, 가능하면 배열로도 받을 수 있을 것
- 컴포넌트의 각 간격에 대해 고정적/상대적 여부
- 상대적인 경우 어느 기준에 의존하는지
- 사용자와의 상호작용을 위한 기능

**Comment by nanana3679** [comment]
@gee2450

**Reply by gee2450** [reply]
확인했습니다. 
이전에는 이슈에 어떤 정보를 기입해야하는지 정확히 알지 못해, 기능만 기입하고 테스트 화면만 추가했는데, 저런 내용을 적으면, 코드를 자세히 분석할 필요 없이, 빠른 시간안에 리뷰가 가능할 것 같아 좋은 것 같습니다!

현재 이슈, 다음 컴포넌트 작업때 참고하여 내용 추가하겠습니다~!
